exercises/concept/bird-watcher/bird_watcher.go

```diff
@@ -0,0 +1,21 @@
+package birdwatcher
+
+// TotalBirdCount return the total bird count by summing
+// all the items in the given array
+func TotalBirdCount(birdsPerDay []int) int {
+	panic("Please implement the TotalBirdCount() function")
+}
+
+// BirdsInWeek returns the total bird count by summing
+// only the items belonging to the given week
+func BirdsInWeek(birdsPerDay []int) int {
+	panic("Please implement the BirdsInWeek() function")
+
+}
+
+// FixBirdCountLog returns an array after correcting
+// the bird counts for alternate days in the given array
```

Comment:
Maybe leave out the implementation detail about arrays?

```suggestion
// FixBirdCountLog returns the bird counts after correcting
// the bird counts for alternate days
```